tests/macros/shared/src/main/scala/scala/macros/tests/scaladays/main.scala

```diff
@@ -0,0 +1,12 @@
+package scala.macros.tests
+package scaladays
+
+import scala.macros._
+
+class main extends MacroAnnotation {
+  inline def apply(defn: Any): Any = meta {
```

Comment:
We can do better than `Any => Any`. See the discussion about this here: https://github.com/scalamacros/scalamacros/issues/6.